gtest-parallel

```diff
@@ -117,19 +118,23 @@ class Outputter(object):
 
 
 class Task(object):
```

Comment:
Could you add a comment describing what this class does? (It should already have been there. This patch just makes the problem a bit worse.)